test/utils/setup.js

```diff
@@ -1,3 +1,4 @@
+require('@babel/register')({ extensions: ['.js', '.jsx', '.ts', '.tsx'] });
```

Comment:
Very unfortunate that typescript tries very hard to offere incremental adoption but most of the tooling doesn't. This doesn't seem right to me but as long as it works, we use it.